libsel4/tools/syscall_stub_gen.py

```diff
@@ -1069,6 +68,8 @@ def process_args():
     wsizegroup.add_argument("-c", "--cfile", dest="cfile",
```

Comment:
The naming is a bit unfortunate now, because this is not related to "C". Can't we drop this, because Kbuild is no longer in use and CMake knows `KernelWordSize`/`WORD_SIZE`, so it could pass this here. Or pass `sel4arch` and there is an internal table then that deductes  the word size

Reply:
As for now, I've renamed this to `"-c", "--conf-file", dest="conf_file"`.
The short option, which is used in `libsel4/CMakeLists.txt` is the same, so there was no need to change it.